rest-api-spec/src/main/resources/rest-api-spec/test/indices.split/30_copy_settings.yml

```diff
@@ -64,6 +65,7 @@
   - match: { copy-settings-target.settings.index.merge.scheduler.max_thread_count: "2" }
   - match: { copy-settings-target.settings.index.blocks.write: "true" }
   - match: { copy-settings-target.settings.index.routing.allocation.include._id: $node_id }
+  - match: { copy-settings-target.settings.index.replication.type: "SEGMENT" }
```

Comment:
Thanks, could we also add this test for `shrink` and `clone`? And add another test case that an error will be thrown   if users set the replication type setting explicitly when calling the shrink/clone/split API?(for this case, version check is needed)

Reply:
> And add another test case that an error will be thrown if users set the replication type setting explicitly when calling the shrink/clone/split API

i added UT tests at 
https://github.com/opensearch-project/OpenSearch/blob/81aec5e113bd19a9228133ecd671a7a021421eba/server/src/test/java/org/opensearch/action/admin/indices/shrink/TransportResizeActionTests.java#L178-L197

Reply:
> could we also add this test for shrink and clone

@gaobinlong  also added in yamlRestTest at shrink/clone/split with noraml/exception case at

https://github.com/opensearch-project/OpenSearch/pull/16076/commits/ba11329d2b272b16949afcecdb7df5a3890644ca

Reply:
Thanks, yaml tests are needed for any API change, they don't conflict with the unit tests.